release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -4000,4 +4000,41 @@ $ oc adm release info 4.10.59 --pullspecs
 [id="ocp-4-10-59-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-60"]
+=== RHBA-2023:3217 - {product-title} 4.10.60 bug fix and security update
+
+Issued: 2023-05-03
```

Comment:
Is this date right? It was a few weeks ago. Per the advisories, looks like it should be today's date?